.vscode/launch.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
I don't know if it makes sense for the vscode config files to be in the repository, what do you think @avelino?

Reply:
@walison17 i dont think this should be a part of the source code of the project

Reply:
It should not be included (if we accept from vscode, we will have to accept from other editors), we have devcontainer to help those who are having difficulties setting up the environment/contributing.

We should put `.vscode` in the **gitignore**.